Tests/XcodeGenCoreTests/ArrayExtensionsTests.swift

```diff
@@ -0,0 +1,25 @@
+import XCTest
+@testable import XcodeGenCore
+
+class ArrayExtensionsTests: XCTestCase {
```

Comment:
Another thing the SortedArray does is sort the elements it's initialised with if we could test that too

Reply:
Done